src/App.js

```diff
@@ -1,12 +1,181 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
+import InputField from "./componenten/InputField";
+import Counter from "./componenten/Counter";
+import Button from "./componenten/Button";
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+    const [strawberries, setStrawberries] = useState ( 0 );
+    const [bananas, setBananas] = useState ( 0 );
+    const [apples, setApples] = useState ( 0 );
+    const [kiwis, setKiwis] = useState ( 0 );
+
+    const [formState, setFormState] = useState ( {
+        firstname: '',
+        lastname: '',
+        age: 0,
+        zipcode: '',
+        deliveryFrequency: 'week',
+        deliveryTimeslot: 'daytime',
+        remark: '',
+        agreeTerms: false,
+    } )
+
+    function handleFormChange(e) {
+        const inputName = e.target.name;
+        const inputValue = e.target.type === 'checkbox' ? e.target.checked : e.target.value;
+
+        setFormState ( {
+            formState,
+            [inputName]: inputValue,
+        } )
+    }
+
+    function resetFruits() {
+        setStrawberries ( 0 );
+        setBananas ( 0 );
+        setApples ( 0 );
+        setKiwis ( 0 );
+    }
+
+    function handleSubmit(e) {
+        e.preventDefault ();
+        console.log ( formState );
+        console.log ( `Fruitmand bestelling - aardbeien: ${strawberries}, bananen: ${bananas}, appels: ${apples}, kiwi's: ${kiwis}` );
+    }
+
+    return (
+        <>
+            <h1>Fruitmand bezorgservice</h1>
+            <section className="fruit-counters">
+                <article>
+                    <h2>🍓 Aardbeien</h2>
+                    <Counter
+                        fruitCount={strawberries}
+                        setFruitCount={setStrawberries}
+                    />
+                </article>
```

Comment:
Ik zou `article` en `h2` verplaatsen naar het Counter component, want dat zie ik bij elk fruitsoort terugkomen en kun je dus weer hergebruiken door het te verplaatsen naar het component

Reply:
De <h1>fruitmand<h1> staat in het rode vak bij jou als verandering op regel 7 en bij mij op regel 49. Is dat er dus dat het hele gedeelte naar counter kan. Article en h2 begint onder fruitmand. Misschien kijk ik verkeerd of begrijp ik het niet helemaal. Moeten de regelnummers comments niet corresponderen aan  de huiswerk gemaakte opdracht? Ik hoop dat je mij dit kan uitleggen en anders als je tijd hebt van de week of volgende week video bellen. Gr Angelique